tests/ui/associated-types/associated-types-in-ambiguous-context.stderr

```diff
@@ -21,10 +21,17 @@ error[E0223]: ambiguous associated type
 LL | type X = std::ops::Deref::Target;
    |          ^^^^^^^^^^^^^^^^^^^^^^^
    |
-help: if there were a type named `Example` that implemented `Deref`, you could use the fully-qualified path
+help: use the fully-qualified path
```

Comment:
Why does the output change?

Reply:
Due to the `compiler/rustc_hir_analysis/src/astconv/mod.rs` change.
Previously, impl visibility was used, but impl visibility is meaningless (and no longer encoded with this PR), trait visibility was actually intended there.